Accesos a maint

SpiritualShadow opened the pull request:
## What Does This PR Do
Le da acceso a maintenance al RD, CMO, a los científicos y a los robotistas

## Why It's Good For The Game
El acceso a maint es un acceso muy básico que tienen hasta los civiles que es útil para pillar heridos o en el caso de ciencias subir RnD con algunos objetos que se pueden encontrar allí. 

## Changelog
:cl:
add: Acceso a maintenance para CMO y ciencias
/:cl:

SpiritualShadow committed:
Accesos a maint

Le da acceso a mantenimiento al RD, CMO, Robotista y Científico

SpiritualShadow committed:
Accesos a maint

Da acceso a maint al RD, CMO, Científicos y Robotistas

SpiritualShadow committed:
Revert "Accesos a maint"

This reverts commit 3e03097c2abb263bb96e340e438c2642ea2e2023.

SagaStars commented:
**COUGHS** Falta el magistrado **COUGHS**
![48154495-D5D9-4A04-BCEA-9B8813CA4E0C](https://user-images.githubusercontent.com/55407528/72673190-50b9ee80-3a3d-11ea-93db-7bd0a56837c5.jpeg)

SpiritualShadow committed:
Loadout para robotistas

Le da todas las ropas de ciencias del loadout que faltaban al robotista (Beret, long labcoats, winter coats)
